libraries/swagger-ui/.agignore

```diff
@@ -0,0 +1 @@
+dist/
```

Comment:
Contrib modules or other repository code cannot be added to our repo.  Please modify composer to pull in any needed dependencies.

Reply:
Will update the composer.* files to reflect this. Thanks @cnovak